src/main/resources/swagger/ga4gh-tool-discovery.yaml

```diff
@@ -235,6 +235,38 @@ paths:
           description: The tool can not be output in the specified type.
           schema:
             $ref: '#/definitions/Error'
+  /tools/{id}/versions/{version_id}/{type}/files:
+    get:
+      summary: Get an array of objects that contain the relative path and file type.  Intended for use with the /tools/{id}/versions/{version_id}/{type}/descriptor/{relative_path} endpoint.
+      tags:
+        - GA4GH
+      parameters:
+        - name: type
+          required: true
+          in: path
+          description: The output type of the descriptor. If not specified it is up to the underlying implementation to determine which output type to return. Plain types return the bare descriptor while the "non-plain" types return a descriptor wrapped with metadata. Allowable values are "CWL" and "WDL".
+          type: string
+        - name: id
+          in: path
+          description: A unique identifier of the tool, scoped to this registry, for example `123456`
+          required: true
+          type: string
+        - name: version_id
+          in: path
+          required: true
+          type: string
+          description: An identifier of the tool version for this particular tool registry, for example `v1`
+      responses:
+        '200':
+          description: The array of File JSON responses.
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/ToolFile'
+        '404':
+          description: The tool can not be output in the specified type.
```

Comment:
This might have been copied and not changed

Reply:
I believe the error is for when somebody puts an unknown descriptor type in the path parameter.  Even though the tool exists, the tool can not be output in the specified type.  Using WDL on a CWL tool on the other hand does not trigger a 404, only a 204.